spec/models/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before :each do
+    @user = User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher')
+  end
+
+  it 'name must not be blank' do
+    @user.name = nil
+    expect(@user).to be_valid
+  end
```

Comment:
- [ ] For this test to pass, you will to update your user.rb model with validation "`Name must not be blank.`"  please make sure the validation is added.